### PR TITLE
Release feature overlap - stop

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/com1DFAPy/com1DFA.py [hunk 1]
```diff
@@ -1269,7 +1269,16 @@ def prepareArea(releaseLine, dem, relThList=''):
...
-            Raster = Raster + rast
+            ind1 = Raster > 0
+            ind2 = rast > 0
+            indMatch = np.logical_and(ind1, ind2)
+            try:
+                message = 'Release area features are overlaping - this is not allowed'
+                assert indMatch.any() == False, message
+                Raster = Raster + rast
+            except AssertionError:
+                log.error('Release area features are overlaping - this is not allowed')
```
Comment (fso42) [inline avaframe/com1DFAPy/com1DFA.py:1280]: This is redundant, why not use the message you provide with the assert check?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFAPy/com1DFA.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFAPy/com1DFA.py]: done

File: avaframe/com1DFAPy/com1DFA.py [hunk 2]
```diff
@@ -569,6 +565,10 @@ def initializeParticles(cfg, relRaster, dem, logName=''):
         # plt.plot(particles['y'], particles['z'], 'k+')
         # plt.show()
     else:
+        # initiate random generator
+        seed = 12345
```
Comment (fso42) [inline avaframe/com1DFAPy/com1DFA.py:569]: This is a setting, please move to a dev flag at top of file or to the ini file...

**Review by fso42 — CHANGES_REQUESTED** [review]

**Review by fso42 — APPROVED** [review]

File: avaframe/com1DFAPy/com1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFAPy/com1DFA.py]: done

**Pull request merged**